papaparse.js

```diff
@@ -1432,4 +1432,11 @@
 	{
 		return typeof func === 'function';
 	}
+
+	// if requirejs is available, return Papa
+	if (typeof define === "function" && define.amd) {
```

Comment:
For consistency, let's use isFunction (defined just above your edit, actually). And if `define` is defined on the `window` object (accessible as `global` inside the Papa closure), let's explicitly say `global.define` to make it clear.